storage/src/time_stamp_comparator.cpp

```diff
@@ -0,0 +1,70 @@
+#include "rocksdb/time_stamp_comparator.h"
+#include "endianness.hpp"
+#include "assertUtils.hpp"
+
+namespace concord {
+namespace storage {
+namespace rocksdb {
+
+#ifdef USE_ROCKSDB
+
+using namespace ::rocksdb;
+
+class Lexicographic64TsComparator : public Comparator {
+ public:
+  Lexicographic64TsComparator() : Comparator(/*ts_sz=*/sizeof(uint64_t)), cmp_without_ts_(BytewiseComparator()) {
+    ConcordAssert(cmp_without_ts_ != nullptr);
+    ConcordAssert(cmp_without_ts_->timestamp_size() == 0);
+  }
+  const char* Name() const override { return "Lexicographic64TsComparator"; }
+  void FindShortSuccessor(std::string*) const override {}
+  void FindShortestSeparator(std::string*, const Slice&) const override {}
+  int Compare(const Slice& a, const Slice& b) const override {
+    int ret = CompareWithoutTimestamp(a, b);
```

Comment:
should this be :     int ret = -CompareWithoutTimestamp(a, b);
As we want descending sorting.

Reply:
Discussed this, so resolving this comment.